test/stack-layers_test.js

```diff
@@ -211,10 +211,11 @@ describe('stack layers function', function() {
         {tag: 'rect', attr: {x: -50, y: -50, width: 1100, height: 1100, fill: '#fff'}},
         {tag: 'use', attr: {'xlink:href': '#id_top_drl1'}},
         {tag: 'use', attr: {'xlink:href': '#id_top_drl2'}},
+        {tag: 'g', attr: {}, children: [0, 1, 2]},
         {
           tag: 'mask',
           attr: {id: 'id_top_mech-mask', fill: '#000', stroke: '#000'},
-          children: [0, 1, 2]
+          children: [3]
```

Comment:
Hey sorry for the delay; work's been crazy and I've managed to come down with the flu. Fun times.

Looks like you just need to change the line below to match the changes you made here:

```
expect(result.defs).to.contain(values[4])
```